Documentation filemanager s3 (#402)
OntimizeWeb/docs: _ontimize_web_ngx_filemanager/general/02-installation.md

```html
<o-filemanager-table service="customers" workspace-key="ID_DMS_DOC"></o-filemanager-table>
```

> If you want to use the SDMS file manager engine you need to add the `type` attribute to yours `o-filemanager-table` component. Also you can define your own workspace as the example bellow.

Reviewer: ```suggestion
If you want to use the SDMS file manager engine you need to add the `type='S3'` attribute to yours `o-filemanager-table` component. 

Also you must indicate also the form column name that stores the work space identifier in the workspace-key attribute or define the `workspace-S3` function as the example bellow
```

> `cutomers-home.html`:

Reviewer: ```suggestion
`customers-detail.component.html`:
```

```html
...

> <o-filemanager-table type="S3" service="customers" workspace-key="ID_DMS_DOC" workspaceS3="setWorkspaceS3(form.data)"></o-filemanager-table>

Reviewer: ```suggestion
<o-filemanager-table type="S3" service="customers" [workspace-s3]="setWorkspaceS3"></o-filemanager-table>
```

...
```

> `customers-home.ts`:

Reviewer: ```suggestion
`customers-detail.component.ts`:
```

```typescript
...
  setWorkspaceS3(data: any) {

>     return { name: 'default', data: { id: [1] } };

Reviewer: ```suggestion
    return { name: 'default', data: { id: data['CUSTOMERID'] } };
```
